src/java/org/apache/cassandra/schema/TableMetadata.java

```diff
@@ -770,6 +770,18 @@ public static TableMetadata minimal(String keyspace, String name)
                             .build();
     }
 
+    /**
+     * There is a couple of places in the code where we need a TableMetadata object and don't have one readily available
+     * and know that only the keyspace and name matter. This creates such "fake" metadata. Use only if you know what
+     * you're doing.
+     */
+    public static TableMetadata minimal(String keyspace, String name, TableId tableId)
```

Comment:
this is used in `test/unit/org/apache/cassandra/service/accord/AccordTestUtils.java`, can we mark this for tests?

```suggestion
    @VisibleForTesting
    public static TableMetadata minimal(String keyspace, String name, TableId tableId)
```